packages/SwingSet/src/kernel/vatManager/manager-nodeworker.js

```diff
@@ -132,9 +132,13 @@ export function makeNodeWorkerVatManagerFactory(tools) {
       parentLog(`sending delivery`, vatDeliveryObject);
       assert(!waiting, X`already waiting for delivery`);
       const pr = makePromiseKit();
+      transcriptManager.startDispatch(vatDeliveryObject);
```

Comment:
yikes! kind of a big bug.

Evidently we don't use these enough to be confident that they're current. I guess that's OK?

Reply:
Yeah, to some extent the nodeworker/subprocess-node manager types are just there to keep me honest and force me to factor things better. I'm working on a much larger refactoring that gets rid of a lot of the redundant code.